homeassistant/components/avea/config_flow.py

```diff
@@ -0,0 +1,18 @@
+"""Config flow for Avea."""
+import avea
```

Comment:
Avea has been commented out in https://github.com/home-assistant/core/blob/35f832bc446b7643efc647c99e118c663723d2f3/requirements_all.txt#L303 which is likely why you are getting this failure. I suspect you need to add an updated version of this file to your PR.

Reply:
This is by design: https://github.com/home-assistant/core/commit/df9de8eb5d23b4c4325ced08354265feb4646aeb#diff-ea94dcf9c98793d02e5142e31614ab5c1d1a65c307373ef9706fbfc948180148

I have no idea what to do. When I activate the package again the "Check all requirements" test fails. But with the commented out package the imports fail?

Reply:
The reason it has been disabled because `bluepy` doesn't install on every platform, so it would fail to set up a test environment. 

This is an interesting case that we haven't encountered before

Reply:
"Interesting" sounds as if I had to choose another idea for my hacktoberfest shirt... :(

What should I do? Should I remove the tests even though I have implemented a config flow?

Reply:
This will still count, as Hacktoberfest counts PRs opened. 

But yeah, I don't really have a solution right now 👀